qiskit_experiments/framework/restless_experiment.py

```diff
@@ -0,0 +1,96 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Restless experiment class."""
+
+from abc import ABC
+from typing import Union
+
+from qiskit.providers import BaseBackend
+from qiskit.test.mock.fake_backend import FakeBackend
+from qiskit_experiments.framework.base_experiment import BaseExperiment
+
+from qiskit_experiments.data_processing.data_processor import DataProcessor
+from qiskit_experiments.data_processing.exceptions import DataProcessorError
+from qiskit_experiments.data_processing import nodes
+
+
+class RestlessEnabledExperiment(BaseExperiment, ABC):
+    """Restless experiment class."""
+
+    def enable_restless(self, rep_delay: float, backend: Union[BaseBackend, FakeBackend]):
```

Comment:
```suggestion
    def enable_restless(self, rep_delay: float, backend: Union[BaseBackend, FakeBackend], override_processor: bool = True):
```
So that users can still provide their own data processor.

Reply:
I modified the code here: [0aac8bd](https://github.com/Qiskit/qiskit-experiments/pull/689/commits/0aac8bd682aeae6e918c8b04d00d89937462b4b3).